GitUI/CommandsDialogs/FormCommit.Designer.cs

```diff
@@ -1033,12 +1034,27 @@ private void InitializeComponent()
             //
             // splitRight.Panel2
             //
+            this.splitRight.Panel2.Controls.Add(this.modifyCommitMessageButton);
             this.splitRight.Panel2.Controls.Add(this.tableLayoutPanel1);
             this.splitRight.Size = new System.Drawing.Size(517, 622);
             this.splitRight.SplitterDistance = 426;
             this.splitRight.TabIndex = 0;
             this.splitRight.TabStop = false;
             //
+            // modifyCommitMessageButton
+            //
+            this.modifyCommitMessageButton.AutoSize = true;
+            this.modifyCommitMessageButton.BackColor = System.Drawing.SystemColors.ButtonFace;
+            this.modifyCommitMessageButton.FlatStyle = System.Windows.Forms.FlatStyle.Flat;
+            this.modifyCommitMessageButton.ForeColor = System.Drawing.SystemColors.HotTrack;
+            this.modifyCommitMessageButton.Location = new System.Drawing.Point(185, 55);
+            this.modifyCommitMessageButton.Name = "modifyCommitMessageButton";
+            this.modifyCommitMessageButton.Size = new System.Drawing.Size(186, 25);
+            this.modifyCommitMessageButton.TabIndex = 9;
+            this.modifyCommitMessageButton.Text = "Modify this special commit m&essage";
```

Comment:
It feels we're creating a new terminology here. I couldn't find any references to "special commit" (or "normal commits" for that matter).
Will it be less clear if we word as follows?
```suggestion
            this.modifyCommitMessageButton.Text = "Modify the commit m&essage";
```

Reply:
It is a "special (commit) _message_". I understand your objection against introducing new terminology. The new tooltip describes it clear enough.